crates/chain/tests/common/tx_template.rs

```diff
@@ -49,11 +49,11 @@ impl TxOutTemplate {
 }
 
 #[allow(dead_code)]
-pub fn init_graph<'a>(
-    tx_templates: impl IntoIterator<Item = &'a TxTemplate<'a, BlockId>>,
-) -> (TxGraph<BlockId>, SpkTxOutIndex<u32>, HashMap<&'a str, Txid>) {
+pub fn init_graph<'a, A: Anchor + Copy + 'a>(
```

Comment:
Currently, all our `Anchor` implementations impl `Copy`, but that might not always be the case. I think we should loosen the constraint to be `A: Anchor + Clone + 'a`.

Reply:
Is this done?

Reply:
Sorry about delay, we are using clone for this now.